contrib/sssd.spec.in

```diff
@@ -572,6 +593,9 @@ autoreconf -ivf
     --with-sssd-user=%{sssd_user} \
     --with-syslog=journald \
     --with-test-dir=/dev/shm \
+%if 0%{?rhel} == 9
```

Comment:
There are build also for rhel 8, so maybe
%if 0%{?rhel} <= 9

Reply:
RHEL8 isn't supported in CI for 'master' branch.